src/main/java/data/Task.java

```diff
@@ -0,0 +1,89 @@
+package data;
+
+public class Task {
```

Comment:
Perhaps Task could be an abstract class since Task is just the superclass of the rest that are instantiated?